app/src/main/java/org/p2p/wallet/home/ui/main/adapter/TokenAdapter.kt

```diff
@@ -37,6 +39,7 @@ class TokenAdapter(
         R.layout.item_token_hidden -> TokenHiddenViewHolder(parent, listener)
         R.layout.item_token_group_button -> TokenButtonViewHolder(parent, listener)
         R.layout.item_banners -> BannersViewHolder(parent, listener)
+        R.layout.item_title -> TitleViewHolder(parent.inflateViewBinding(parent.context, attachToRoot = false))
```

Comment:
inflating can be inside, for the sake of one way creating ViewHolder`s here

Reply:
should be - to be in common approach